aws/table_aws_rds_db_instance_automated_backup.go

```diff
@@ -0,0 +1,310 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/rds"
+	"github.com/aws/aws-sdk-go-v2/service/rds/types"
+
+	rdsv1 "github.com/aws/aws-sdk-go/service/rds"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsRDSDBInstanceAutomatedBackup(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_rds_db_instance_automated_backup",
+		Description: "AWS RDS DB Instance Automated Backup",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("arn"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"DBInstanceAutomatedBackupNotFound"}),
+			},
+			Hydrate: getRDSDBInstanceAutomatedBackup,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listRDSDBInstanceAutomatedBackups,
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "db_instance_identifier", Require: plugin.Optional},
+				{Name: "dbi_resource_id", Require: plugin.Optional},
+				{Name: "status", Require: plugin.Optional},
+			},
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(rdsv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "db_instance_identifier",
+				Description: "The friendly name to identify the DB Instance.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DBInstanceIdentifier"),
+			},
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) for the replicated automated backups.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DBInstanceAutomatedBackupsArn"),
+			},
+			{
+				Name:        "db_instance_arn",
+				Description: "The Amazon Resource Name (ARN) for the automated backups.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DBInstanceArn"),
+			},
+			{
+				Name:        "status",
+				Description: "Specifies the current state of this database.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "allocated_storage",
+				Description: "Specifies the allocated storage size in gibibytes (GiB).",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "availability_zone",
+				Description: "The Availability Zone that the automated backup was created in.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "backup_retention_period",
+				Description: "The retention period for the automated backups.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "backup_target",
+				Description: "Specifies where automated backups are stored: Amazon Web Services Outposts or the Amazon Web Services Region.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "dbi_resource_id",
```

Comment:
```suggestion
				Name:        "dbi_resource_id",
```
Should it be `db_instance_identifier`?

Reply:
In the [API response](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/rds@v1.40.1/types#DBInstanceAutomatedBackup) the property name is `DbiResourceId`